README.md

```diff
@@ -4,13 +4,19 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # beman.exemplar: A Beman Library Exemplar
 
-![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
+<!-- markdownlint-disable -->
+<img src="https://github.com/bemanproject/beman/blob/main/images/logos/beman_logo-beman_library_under_development.png" style="width:5%; height:auto;"> ![Continuous Integration Tests](https://github.com/bemanproject/exemplar/actions/workflows/ci_tests.yml/badge.svg)
+<!-- markdownlint-enable -->
 
 `beman.exemplar` is a minimal C++ library conforming to [The Beman Standard](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_STANDARD.md).
 This can be used as a template for those intending to write Beman libraries.
 It may also find use as a minimal and modern  C++ project structure.
 
-Implements: `std::identity` proposed in [Standard Library Concepts (P0898R3)](https://wg21.link/P0898R3).
+**Implements**: `std::identity` proposed in [Standard Library Concepts (P0898R3)](https://wg21.link/P0898R3).
+
+<!-- markdownlint-disable -->
+**Status**: [Under development and not yet ready for production use.](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md#under-development-and-not-yet-ready-for-production-use)
+<!-- markdownlint-enable -->
```

Comment:
Why do we need to ignore lint rule here?

```suggestion
**Status**: 
[Under development and not yet ready for production use.](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md#under-development-and-not-yet-ready-for-production-use)
```

Reply:
Because it complains about long line. Which is not actually true, as the rendered version of this line is not so big!

Reply:
is there a setting on the lint that can ignore the long lines or turn off that check altogether?

Reply:
I change the used syntax, and I have shorter lines